src/Wallabag/CoreBundle/Tests/Controller/ConfigControllerTest.php

```diff
@@ -65,12 +65,7 @@ public function dataForUpdateFailed()
             array(array(
                 'config[theme]' => 0,
                 'config[items_per_page]' => '',
-                'config[language]' => 'fr_FR',
```

Comment:
I changed `fr` to `en` to not change all test where we looked for english text. 
I removed the test where language is empty, because it failed. And as you didn't write a test where `config[theme]` is empty, I removed this one. Don't know if I'm clear.